paper-checkbox.html

```diff
@@ -165,7 +165,7 @@
       this.checked = !this.checked;
 
       if (this.checked !== old) {
-        this.fire('iron-change');
+        this.fire('change');
```

Comment:
Hmm, all of `paper-radio-button`/`paper-checkbox'`/paper-toggle-button` fire `iron-change` here, and that's what they used to do before the port.

I think what's more likely is that `_onMouseDown` and `_onKeyDown` should fire `change` (looking at the 0.5 `paper-radio-button`, it fired `change` inside `onTap`.